vmdb/app/models/ext_management_system.rb

```diff
@@ -92,7 +92,13 @@ def self.supported_types_and_descriptions_hash
   include AuthenticationMixin
   include Metric::CiMixin
   include AsyncDeleteMixin
+  include ReservedMixin
 
+  attr_via_reserved :realm
+  attr_via_reserved :security_protocol
```

Comment:
Yes, reserves are a downstream-only mechanism to avoid having schema
changes in patch releases.  Upstream should not have reserves based code
unless this commit is meant for backporting.
 On Apr 6, 2015 11:15 AM, "Greg Blomquist" <notifications@github.com> wrote:

> In vmdb/app/models/ext_management_system.rb
> <https://github.com/ManageIQ/manageiq/pull/2503#discussion_r27803104>:
>
> >
> > +  attr_via_reserved :realm
> > +  attr_via_reserved :security_protocol
>
> I think we only want to do this for patch releases. Since this is a real
> release, these should be real columns with a migration.
>
> @Fryguy <https://github.com/Fryguy>, does that sound right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/manageiq/pull/2503/files#r27803104>.
>